ADAL/src/urlprotocol/ADURLProtocol.m

```diff
@@ -136,7 +136,9 @@ + (BOOL)canInitWithRequest:(NSURLRequest *)request
     //all traffic while authorization webview session is displayed for now.
     if ( [[request.URL.scheme lowercaseString] isEqualToString:@"https"])
     {
-        AD_LOG_VERBOSE(_reqContext(request).correlationId, @"%@ - host: %@", @"+[ADURLProtocol canInitWithRequest:] handling host", [request.URL host]);
+        AD_LOG_VERBOSE(_reqContext(request).correlationId, @"%@ - host: %@", @"+[ADURLProtocol canInitWithRequest:] handling host", [ADAuthorityUtils isKnownHost:request.URL] ? [request.URL host] : @"unknown host");
+        AD_LOG_VERBOSE_PII(_reqContext(request).correlationId, @"%@ - host: %@", @"+[ADURLProtocol canInitWithRequest:] handling host", [request.URL host]);
```

Comment:
nit: why @"+[ADURLProtocol canInitWithRequest:] handling host" is a separate argument?

Reply:
fixed